packages/web-components/src/components.d.ts

```diff
@@ -1540,6 +1540,19 @@ export namespace Components {
          */
         "width"?: string;
     }
+    /**
+     * @componentName Service List
+     * @maturityCategory use with caution
```

Comment:
this should be set to ` * @maturityCategory caution` and that _should_ fix it's maturity indicator not showing in storybook